python/sdk/test/integration_test.py

```diff
@@ -838,17 +838,125 @@ def test_standard_transformer_with_multiple_feast(
     exp_resp = {
         "instances": {
             "columns": [
-                "rank",
-                "driver_id",
-                "customer_id",
-                "merlin_test_driver_features:test_int32",
-                "merlin_test_driver_features:test_float",
-                "merlin_test_driver_features:test_double",
+            "rank",
+            "driver_id",
+            "customer_id",
+            "merlin_test_redis_driver_features:completion_rate",
+            "merlin_test_redis_driver_features:cancellation_rate",
+            "merlin_test_bt_driver_features:rating"
             ],
-            "data": [[0, "1234", 1111, -1, 0, 0], [1, "5678", 1111, -1, 0, 0]],
+            "data": [
+            [
+                0,
+                "driver_1",
+                1111,
+                0.85,
+                0.15,
+                4.2
+            ],
+            [
+                1,
+                "driver_2",
+                1111,
+                0.6,
+                0.4,
+                4.2
+            ]
+            ]
         }
     }
 
+
+    assert resp.json()["instances"] == exp_resp["instances"]
+    merlin.undeploy(v)
+
+@pytest.mark.feast
+@pytest.mark.integration
+def test_standard_transformer_with_multiple_feast_with_source(
```

Comment:
Will it test the direct storage approach?

Reply:
yes